src/monitor-plan-comment-workspace/monitor-plan-comment.service.ts

```diff
@@ -110,27 +111,48 @@ export class MonitorPlanCommentWorkspaceService {
 
   async importComments(
     commentData: MonitorPlanCommentBaseDTO[],
+    locationIds: string[],
     userId: string,
-    monitorPlanId: string,
-    isImport: boolean,
     trx?: EntityManager,
   ) {
     return settlePromises(
-      commentData.map(async comment => {
-        const monitorPlanComment = await this.getCommentsByPlanIdCommentBD(
-          monitorPlanId,
-          comment.monitoringPlanComment,
-          comment.beginDate,
-          trx,
+      commentData.map(async (comment) => {
+        const res = await this.entityManager.query(
+          'select * from camdecmpswks.get_plan_by_comment_begin_and_end_date($1,$2,$3)',
+          [locationIds, comment.beginDate, comment.endDate],
+        );
+        console.log("plantsIds!")
+        const planIds = res.map((row) => row.mon_plan_id);
+        console.log(planIds)
```

Comment:
Can you remove these console logs?
```suggestion
        const planIds = res.map((row) => row.mon_plan_id);
```